mcs/class/corlib/corlib.dll.sources

```diff
@@ -120,7 +120,7 @@ System/MonoType.cs
 System/MonoTouchAOTHelper.cs
 System/MulticastDelegate.cs
 System/NullConsoleDriver.cs
-System/Nullable.cs
+../../../external/corefx/src/Common/src/CoreLib/System/Nullable.cs
```

Comment:
this one is not so simple since it is accessed by the runtime, not sure if it's worth replacing at this point.